app/app_xdg.go

```diff
@@ -24,6 +26,25 @@ func (app *fyneApp) OpenURL(url *url.URL) error {
 	return cmd.Run()
 }
 
+func (app *fyneApp) SendNotification(n *fyne.Notification) {
+	conn, err := dbus.SessionBus() // shared connection, don't close
+	if err != nil {
+		fyne.LogError("Unable to connect to session D-Bus", err)
+		return
+	}
+
+	appName := ""       // considered fyne.CurrentApp().UniqueID() but really would need a prettier name
```

Comment:
I think we should not use an empty string here.
Either use the `UniqueID()` (ugly or not) or use a constant like `FyneApp` for now.